pkgs/by-name/gp/gpu-screen-recorder/package.nix

```diff
@@ -68,6 +68,11 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.mesonBool "nvidia_suspend_fix" false)
   ];
 
+  postPatch = ''
+    substituteInPlace extra/gpu-screen-recorder.service \
+      --replace-fail "ExecStart=gpu-screen-recorder" "ExecStart=$out/bin/gpu-screen-recorder"
+  '';
+
```

Comment:
This change is included in the [by-name commit](https://github.com/NixOS/nixpkgs/pull/367552/commits/1c3fd94fd1e0e10b73fa636293330d38812026f1)

Reply:
Nice catch! While we are here, mind reviewing #367548? :)